core/tests/webdriverio_utils/waitFor.js

```diff
@@ -280,6 +280,15 @@ var clientSideRedirection = async function(
   await waitForCallerSpecifiedConditions();
 };
 
+var condition = async function(
```

Comment:
Don't expose this. We only want a specific set of waitFor building blocks for developers to use.

Reply:
Sorry, I don't see how to do it given that waitForNonEmptyText depends on the action.js

Reply:
See above -- I don't think you actually need the strict dep on action.js.